cordova-js-src/exec.js

```diff
@@ -41,6 +41,20 @@ var execProxy = require('cordova/exec/proxy');
  */
 module.exports = function (success, fail, service, action, args) {
 
+    // Handle the case when we have an old version of splashscreen plugin to avoid the API calls failures
+    if (service === 'SplashScreen') {
+        var pluginsVersions = require("cordova/plugin_list").metadata;
+        var splashscreenVersion = pluginsVersions['cordova-plugin-splashscreen'];
+        var MIN_SPLASHSCREEN_SUPPORTED_VER = 4;
+        if (splashscreenVersion && parseInt(splashscreenVersion[0], 10) < MIN_SPLASHSCREEN_SUPPORTED_VER) {
+            console.log('Warning: cordova-plugin-splashscreen has been hooked into for compatibility reasons. Update the plugin to version >= 4.');
```

Comment:
Consider updating the message to something like: "A more recent version of cordova-plugin-splashscreen has been hooked into..."